tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -421,7 +421,7 @@ Feature: dav-versions
       | new         | Brian | /testshare |
 
 
-  Scenario: receiver tries to get file versions of unshared file from the sharer
+  Scenario: sharee tries to get file versions of file not shared by the sharer
```

Comment:
```suggestion
  Scenario: sharee tries to get file versions of another user's file
```

Reply:
another thing I don't understand is why we have to share the files step below. We can remove these steps
```
    And user "Alice" has uploaded file with content "textfile0" to "textfile0.txt"
    And user "Alice" has shared file "textfile0.txt" with user "Brian"
    And user "Brian" has accepted share "/textfile0.txt" offered by user "Alice"
```

Reply:
@saw-jan @SagarGi @SwikritiT  what you guys say!  do we need to add share steps in this scenario ?????

Reply:
IMO, should be fine there. This scenario tests the edge case that when textfile0 is shared with Brian, he cannot view the versions of another unshared file of Alice.